lib/src/DatePicker/DatePicker.tsx

```diff
@@ -48,10 +61,16 @@ export class DatePicker extends React.PureComponent<DatePickerProps & WithUtilsP
     minDate: '1900-01-01',
     maxDate: '2100-01-01',
     openToYearSelection: false,
+    year: false,
+    month: false,
   };
 
-  public state = {
-    showYearSelection: Boolean(this.props.openToYearSelection),
+  public state: DatePickerState = {
+    openView: Boolean(this.props.openToYearSelection || this.props.year)
```

Comment:
I think here it should be the prop`openTo` the same as on date time picker. We can left `openToYear` but make a enum instead of 2 bools

Reply:
Maybe it will be nice to have something like 
```
initialView?: 'year' | 'month' | 'day';
maxView?: 'year' | 'month' | 'day';
minView?: 'year' | 'month' | 'day';
```
later it can be scaled to make possible to pick any precision, like pick only month and day or only month, what do you think?

Reply:
Thats seems to be an option. But we definitely need `'year' | 'month' | 'day'` type for the views. Similar to date time picker view.

Reply:
So what is the final decision?

Reply:
The final decision - for now add `openTo` property. The same as in DateTimePicker with enum value `'year' | 'month' | 'day'`. 
Also we don't want to introduce breaking changes so `openToYearSelection` should work as well but marked as deprecated. Then we will remove it. 

About minView and maxView - I am considering. The naming is not really obvious. I am considering something like. `availableView` which will be an `Array<'year' | 'month' | 'day'>`. For month picker something like `availableView=['month']`. But that can be done later

Reply:
Sorry but I still did not understand how to replace ```year``` and ```month``` boolean props.

```openTo``` is about initial view, it's clear.

Reply:
@dmtrKovalenko so what about that year and month props?

Reply:
@OleksiiKukuruza instead of 2 bools describing which options to add I propose to add array of views that will be shown.

So it will be possible to just show month picker and nothing more.

Reply:
@OleksiiKukuruza And please - rebase your branch from up-to-date develop. Cause you're breaking coverage reporter :)